Blog Update - SSH Bastion Host

sshahcodes opened the pull request:
This update is already approved in the web repo - https://github.com/gravitational/web/pull/2838

Preview [here](https://blog-git-sshahcodes-update-ssh-bastion-host-gravitational.vercel.app/blog/ssh-bastion-host/)

sandylcruz commented on pages/ssh-bastion-host/index.mdx:
````diff
@@ -99,25 +99,41 @@ For that to work, create a separate user account for regular users. In this exam
 ```
 
-The examples above will work only if the public SSH keys of your users are copied to both the bastion host and the destination machines, which can be a hindrance. We advocate against using SSH keys and moving to [SSH certificates](/ssh-key-management/) instead. Teleport, the next open source SSH solution we’ll talk about, uses SSH certificates by default and it does not require any configuration  ...
+The examples above will work only if the public SSH keys of your users are copied to both the bastion host and the destination machines, which can be a hindrance. We advocate against using SSH keys and moving to [SSH certificates](/blog/ssh-key-management/) instead. Teleport, the next open source SSH solution we’ll talk about, uses SSH certificates by default and it does not require any configura ...
````
The link should just be `/ssh-key-management` as `/blog/` is part of the base URL

sandylcruz commented on pages/ssh-bastion-host/index.mdx:
```diff
@@ -128,11 +144,13 @@ One interesting feature of Teleport is that it is environment-aware, and makes a
 Teleport supports other protocols in addition to SSH, so the same bastion can be used to access other resources behind NAT, such as Kubernetes clusters or even internal applications via HTTP(s).
 
-Teleport’s own quick start guide includes easy instructions for setting up the bastion, so we won’t be copy-pasting the instructions from there. Just start by [downloading Teleport](https://goteleport.com/download/) for your platform and follow the [quick start guide](https://goteleport.com/docs/quickstart/).
+Teleport’s own quick start guide includes easy instructions for setting up the bastion, so we won’t be copy-pasting the instructions from there. Just start by [downloading Teleport](/teleport/download/) for your platform and follow the [quick start guide](/teleport/docs/quickstart/).
```
This link should be an absolute URL since blog is in a separate repository

sandylcruz commented on pages/ssh-bastion-host/index.mdx:
```diff
@@ -128,11 +144,13 @@ One interesting feature of Teleport is that it is environment-aware, and makes a
-Teleport’s own quick start guide includes easy instructions for setting up the bastion, so we won’t be copy-pasting the instructions from there. Just start by [downloading Teleport](https://goteleport.com/download/) for your platform and follow the [quick start guide](https://goteleport.com/docs/quickstart/).
+Teleport’s own quick start guide includes easy instructions for setting up the bastion, so we won’t be copy-pasting the instructions from there. Just start by [downloading Teleport](/teleport/download/) for your platform and follow the [quick start guide](/teleport/docs/quickstart/).
+
+If you are running on AWS, here's a tutorial on using Teleport as a [Bastion host in AWS](https://goteleport.com/blog/how-to-setup-aws-bastion/).
```
This link can just be `/how-to-setup-aws-bastion`

sshahcodes commented:
@sandylcruz Thanks for pointing out path errors. I've fixed those, but the build is still failing. 
```
Error occurred prerendering page "/ssh-jump-server". Read more: https://nextjs.org/docs/messages/prerender-error
TypeError: image.split is not a function
...
```

sandylcruz commented:
Not really sure why that kept failing to build, but it's fixed now

sshahcodes commented on pages/ssh-bastion-host/index.mdx:
```diff
@@ -1,13 +1,13 @@
 articleTitle: Setting Up an SSH Bastion Host
+description: In this blog post, we’ll go over the installation and configuration of SSH bastion hosts using two open source products.
+title: What is an SSH Bastion? | SSH Bastion host setup
+date: "2022-03-31"
```
```suggestion
date: "2022-04-04"
```

sshahcodes commented:
Thanks, @sandylcruz. I have updated the publish date as well. This can be merged now.
